SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -610,6 +620,16 @@ void USpatialWorkerConnection::ProcessOutgoingMessages()
 				Message->EntityId,
 				&Message->Update,
 				&DisableLoopback);
+
+#if TRACE_LIB_ACTIVE
```

Comment:
If we are tightly coupling these things in gdk code then we need to be careful for worker wrapper changes or other changes to how we deal with the worker sdk.

Reply:
Open for suggestions on how to make this play nice with the worker wrapper.

Reply:
I meant to write before that comment - "note for future". As I don't think this requires a change now. We just need to be careful that we keep track of where our asynchronicity is.

Reply:
Actually I guess this is always on gdk send api and worker send API. Will resolve this is it's just noise rather than review.